babel-preset/configs/hmr.js

```diff
@@ -18,6 +18,10 @@ module.exports = function(options, filename) {
   var transform = filename
       ? './' + path.relative(path.dirname(filename), transformPath) // packager can't handle absolute paths
       : hmrTransform;
+
+  // Fix the path to use '/' on Windows.
+  transform = transform.replace(/\\/g, '/');
```

Comment:
It should check the path separator first,

```js
if (path.sep === '\\') {
  transform = transform.replace(/\\/g, '/');
}
```

I think we should move this to a module `normalizePath` or something, and use it instead of duplicating the code.